ush/ioda/bufr2ioda/adpupa_prepbufr_2021080100.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
as @RussTreadon-NOAA said, this looks like a test file and needs removed

Reply:
@CoryMartin-NOAA removed, thanks.